format/partition-statistics-spec.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Partition Statistics Spec"
+url: partition-statistics-spec
+toc: true
+disableSidebar: true
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Partition Statistics file format
```

Comment:
For other metadata files, we don't have separate spec documents. Could you incorporate this into the Iceberg spec?

Reply:
ok. Puffin had a separate doc. So, I added it as a separate doc. 

I will move to Iceberg spec itself.